.github/local-development.md

```diff
@@ -0,0 +1,107 @@
+# Local Development
+
+This document gives tips and tricks on how to create the Renovate documentation website locally to add features or fix bugs.
+You can improve this documentation by opening a pull request.
+For example, if you think anything is unclear, or you think something needs to be added, open a pull request!
+
+## Installation
+
+### Prerequisites
+
+You need the following dependencies for local development:
+
+- Git
+- Node.js `^12.13.0 || >=14.15.0`
```

Comment:
Maybe we should use:

```suggestion
- Node.js `^12.13.0`
```

As that is what the GitHub action is using now.

Have we tested that Node 14 will work?

Reply:
We only support latest lts. We should fix the action

Reply:
I will open a PR to fix the action then. 👍